ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-commands.sh

```diff
@@ -5,10 +5,28 @@ set -o errexit
 set -o pipefail
 
 export KUBECONFIG=${SHARED_DIR}/kubeconfig
-
-echo "$(date -u --rfc-3339=seconds) - Configuring image registry with emptyDir..."
-oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Managed","storage":{"emptyDir":{}}}}'
-
+#Create pvc for image registry storage
+if [[ "${STEPUP_IMAGE_REGISTRY_WITH_PVC}" == "yes" ]]; then
+  echo "$(date -u --rfc-3339=seconds) - Setting the image registry storage to use the manual setting storage..."
+  cat >> "${SHARED_DIR}/registry-pvc.yaml" << EOF
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: registry-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+EOF
+  oc patch config.image/cluster -p '{"spec":{"managementState":"Removed","storage":null}}' --type=merge &&
```

Comment:
This should be first time to setup registry, the default status is Removed already